index.rst

```diff
@@ -1,15 +1,12 @@
 MoveIt 2 Documentation
 ======================
 
-MoveIt 2 is the robotics manipulation platform for ROS 2. For ROS 1, see `MoveIt 1 <https://ros-planning.github.io/moveit_tutorials>`_.
+Welcome to unified MoveIt documentation, which includes tutorials, how-to-guides, core concepts, and more.
 
-.. image:: doc/tutorials/quickstart_in_rviz/rviz_plugin_head.png
-   :width: 700px
+MoveIt 2 is the robotics manipulation platform for ROS 2, which had its first release in 2019. For ROS 1 documentation, see `MoveIt 1 tutorials <https://ros-planning.github.io/moveit_tutorials>`_.
```

Comment:
[nit] swap these so the text "MoveIt 2 is the robotics manipulation platform for ROS 2, which had its first release in 2019...." is what would show up in a google search.

Also, we could still use a much better description of what Moveit is and why someone would use it.  "the robotics manipulation platform" is really vague imo.

Reply:
I think the important part for search results is that this is the unified documentation with X, Y, Z. I would like to keep the order as-is.

What MoveIt is, is less important here. That's more the topic of the MoveIt homepage. I have expanded the description of what MoveIt is per your suggestion.